### ISSUE TITLE
Commands unexpectedly captured in text editing

### ISSUE THREAD

**Issue by Biacco42** [ueokande/vim-vixen#79]
### I'm opening this issue because:

- [x] I'll report a bug
- [ ] I'll propose a new feature

### Description

Commands unexpectedly captured in text editing.  
Especially I noticed that at Twitter's tweeting view.

#### Steps to Reproduce

1. Open http://twitter.com
1. Press tweet button
1. Edit text
1. Occur unexpected command recognition

#### System configuration

- Operating system: macOS Sierra
- Firefox version: 57.0b8
- Vim-Vixen version: 0.1

#### Console logs

Any relevant log in developer tools:

I noticed the Twitter's tweet view composed by `div` tag. Not `input`. Maybe affects.

```html
<div aria-labelledby="tweet-box-global-label" name="tweet" id="tweet-box-global" class="tweet-box rich-editor" spellcheck="true" role="textbox" aria-multiline="true" data-placeholder-default="いまどうしてる？" data-placeholder-poll-composer-on="質問する" data-placeholder-reply="返信をツイート" dir="ltr" aria-autocomplete="list" aria-expanded="false" aria-owns="typeahead-dropdown-5" contenteditable="true">nnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnnn<br></div>
<div class="RichEditor-pictographs" aria-hidden="true"></div>    
```

**Comment by ueokande** [comment]
That is duplicated in #53. This problem will be fixed in v0.2!

**Comment by Biacco42** [comment]
So sorry. I missed.
